shared/constants/metametrics.js

```diff
@@ -154,6 +154,26 @@
  *  the page view
  */
 
+/**
+ * @typedef {Object} Traits
+ * @property {string} [LEDGER_CONNECTION_TYPE] - when ledger live connnection type is changed we identify the ledger_connection_type trait
+ * @property {string} [NETWORKS_ADDED] - when user modifies networks we identify the networks_added trait
+ * @property {string} [NUMBER_OF_ACCOUNTS] - when identities change, we identity the new number_of_accounts trait
+ * @property {string} [THREE_BOX_ENABLED] - when 3box feature is toggled we identify the 3box_enabled trait
+ */
```

Comment:
```suggestions
/**
 * @typedef {Object} Traits
 * @property {string} [LEDGER_CONNECTION_TYPE] - when ledger live connnection
 *  type is changed we identify the ledger_connection_type trait
 * @property {string} [NETWORKS_ADDED] - when user modifies networks we
 *  identify the networks_added trait
 * @property {string} [NUMBER_OF_ACCOUNTS] - when identities change, we
 *  identity the new number_of_accounts trait
 * @property {string} [THREE_BOX_ENABLED] - when 3box feature is toggled we
 *  identify the 3box_enabled trait
 */
```
see previous comment  about the 80 character column limit for comments.

Reply:
Can we update our linting automation to fix/enforce this automatically?

Reply:
(not in this PR, this is just a general question to Brad)

Reply:
I agree with @danjm, my default prettier formatting is 150 chars (I have a huge screen) and I usually run yarn lint:fix before I commit. So if linting can fix/enforce this, it will be great.

Reply:
fixed [here](https://github.com/MetaMask/metamask-extension/pull/14069/commits/5b895a85aa6d5a6978fa80e494777bdb5d902941)